exercises/armstrong-numbers/lib/example.dart

```diff
@@ -6,7 +6,7 @@ class ArmstrongNumbers {
   bool isArmstrongNumber(final int number) {
     final String numberAsString = number.toString();
     final int numOfDigits = numberAsString.length;
-    int sum = 0;
+    num sum = 0;
```

Comment:
Did this have to change because pow returns num? Does the fact that we're comparing `sum` to `number` and the two are different data types?

Reply:
Correct, pow returns `num` and so either we do a cast with `as` or we change the type of `sum`.

Since the values are the same, the different data types don't matter in this case. Could it be confusing to users? Yeah, probably. Would the recommendation be to change number to be of type `num`? Would that require us to change the parameter name to something else?

Reply:
Might be a good idea to change it to `num`. I think it would be beneficial for students to understand the different data types available to them.

I think the name `number` can remain. I don't know what else you could call the parameter that would be intuitive other than `number`.

Reply:
I can rename it `input` and change the type to `num`.